components/library/tntnet/Solaris/http-tntnet.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0"?>
+<!--
+CDDL HEADER START
+
+This file and its contents are supplied under the terms of the
+Common Development and Distribution License ("CDDL"), version 1.0.
+You may only use this file in accordance with the terms of version
+1.0 of the CDDL.
+
+A full copy of the text of the CDDL should have accompanied this
+source.  A copy of the CDDL is also available via the Internet at
+http://www.illumos.org/license/CDDL.
+
+CDDL HEADER END
+-->
+
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+    Copyright (c) 2007, 2011, Oracle and/or its affiliates. All rights reserved.
+    Copyright 2011 EveryCity Ltd. All rights reserved.
+    Copyright (c) 2016 Jim Klimov
+-->
+
+<service_bundle type='manifest' name='tntnet'>
+
+<service
+	name='network/http/tntnet'
```

Comment:
Please, name it network/http and call instance tntnet to be consistent with other http services (nginx comes from EC and was a bad example)